src/Confluent.SchemaRegistry/Rest/DataContracts/SchemaType.cs

```diff
@@ -0,0 +1,40 @@
+// Copyright 2020 Confluent Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+// Refer to LICENSE for more information.
+
+
+namespace  Confluent.SchemaRegistry
+{
+    /// <summary>
+    ///     Enumerates the types of schema supported by Schema Registry.
+    /// </summary>
+    public enum SchemaType
+    {
+        /// <summary>
+        ///     Avro
```

Comment:
Why are the string representations of these in all upper case when these are not?

Reply:
The string representations are used in the REST API and must be all caps. The enums are capitalized to .net conventions.